examples/plot_dem_hillshade.py

```diff
@@ -0,0 +1,153 @@
+"""
+Create A Hillshade From a Digital Elevation Model Using EarthPy
+===============================================================
+
+Learn how to create a hillshade from a DEM using the EarthPy
+es.hillshade() function.
+"""
+
+###############################################################################
+# Create A Hillshade Layer Using EarthPy
+# ---------------------------------------
+#
+# .. note::
+#       The examples below show you how to use the ``es.hillshade()``
+#       function from EarthPy.
+
+###############################################################################
+# Create a Hillshade from a Digital Elevation Model (DEM)
+# -------------------------------------------------------
+# A hillshade is a 3D representation of a surface. Hillshades are generally
+# rendered in greyscale. The darker and lighter colors represent the shadows
+# and highlights that you would visually expect to see in a terrain model.
+# Hillshades are often used as an underlay in a map, to make the data appear
+# more 3-Dimensional and thus visually interesting. This vignette will show
+# you how to create a hillshade from a DEM using EarthPy. It will highlight
+# how to adjust the sun's azimuth, altitude and other settings that will impact 
+# how the hillshade shadows are modeled in the data.
+#
+# The hillshade function is a part of the spatial module in EarthPy.
+
+###############################################################################
+# Import Packages
+# ----------------
+#
+# You will need several packages to get started. You will use Rasterio to
+# open up the DEM file needed to create the hillshade layer.
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+import rasterio as rio
+
+# Download the data needed for this vignette
+data = et.data.get_data("vignette-elevation")
+
+####################################################################################
+# Open up the DEM
+# ---------------
+# To begin, open your DEM layer as a numpy array using Rasterio. Below you set all
+# terrain values < 0 to ``nan``. Next, plot the data using ``ep.plot_bands()``.
+
+# Set the home directory and get the data for the exercise
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+dtm = "data/vignette-elevation/pre_DTM.tif"
+
```

Comment:
@nkorinek i do'nt see your changes. i also see old text her that i had changed previously. things like rasterIO and plott the data. did you update from this branch before pushing changes? and did you add figsize to the plots?

Reply:
@lwasser I made a few changes before pulling from your commit, so sorry! That may have messed up the issues you had fixed. However for me all the plots still have titles and figsize's, do you see that too? My local copy says it's up to date...

Reply:
i got it @nkorinek i think a few things perhaps i missed before and then the fig size looks great now. thank you. this is done. once travis is happy i'll merge :) i messed up in my review here and tried to complete it too quickly.